.github/workflows/ci.yaml

```diff
@@ -68,6 +68,7 @@ jobs:
   integration-test:
     strategy:
       fail-fast: false
+      max-parallel: 6
```

Comment:
Limit the maximum amount of matrix tests running, to to limit the amount of runners used.